recipes/xtr/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+
+
+class XtrConan(ConanFile):
+    name = "xtr"
+    description = \
+        "C++ Logging Library for Low-latency or Real-time Environments"
+    topics = ("xtr", "logging", "logger")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/choll/xtr"
+    license = "MIT"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "enable_exceptions": [True, False],
+        "enable_lto": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "enable_exceptions": True,
+        "enable_lto": False,
+    }
+    generators = "make"
+
+    def requirements(self):
+        self.requires("fmt/7.1.3")
+
+    def validate(self):
+        if self.settings.os not in ("FreeBSD", "Linux"):
+            raise ConanInvalidConfiguration(f"Unsupported os={self.settings.os}")
```

Comment:
To date we have not permitted f-strings [there's 6 commits removing them in PRs](https://github.com/conan-io/conan-center-index/search?q=f-string&type=commits)... I see we have to approvals, are we allowing this now?

Reply:
so-called f-strings are Python 3.6+ : https://www.python.org/dev/peps/pep-0498/
our policy about python versions: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-for-supported-python-versions
TLDR: CCI supports 3.6+, there is no reason to disallow this particular feature in recipes